docs/troubleshooting.md

```diff
@@ -107,37 +107,33 @@ We can instantly see that the Collector has failed to download a kernel driver.
 
 ## Collector Start Up
 
-The vast majority of errors occur during Collector startup, where Collector will
-configure itself, find or download an appropriate kernel driver for the system,
-and then start collecting events. The following diagram describes the main parts
-of this start-up process:
-
-<p align="center">
-  <!-- To edit the diagram, open with app.diagrams.net -->
-  <img src="./imgs/collector-startup.drawio.png" />
```

Comment:
We might want to also remove this from the repo , or is it being used elsewhere?